it/uk/gov/hmrc/incorporatedentityidentificationfrontend/controllers/errorpages/CompanyNumberNotFoundControllerISpec.scala

```diff
@@ -67,15 +58,17 @@ class CompanyNumberNotFoundControllerISpec extends ComponentSpecHelper with Comp
       }
 
       "there is a serviceName passed in the journeyConfig" should {
+        val testJourneyConfig = JourneyConfig(
+          continueUrl = testContinueUrl,
+          pageConfig = PageConfig(optServiceName = Some(testCallingServiceName), deskProServiceId = testDeskProServiceId, signOutUrl = testSignOutUrl),
+          businessEntity = LimitedCompany,
+          businessVerificationCheck = true,
+          regime = "VATC"
```

Comment:
Again, you could potentially use "testRegime" here. I prefer to use constants rather than string literals, but not everyone thinks the same.